### PR TITLE
change save directory to ~/.local/share/pokete, and add migrate.sh to copy the data over

### PR DESCRIPTION
**Pull request by greatericontop**
It's not a great idea to set it to `~/.cache`, because this directory is supposed to be temporary and could be cleared every so often. People also sometimes clear it as it tends to get quite large over time.

### CODE REVIEW

File: release.py [hunk 1]
```diff
@@ -2,5 +2,5 @@
```
Comment (lxgr-linux) [inline release.py:2]:
```suggestion
import os
from pathlib import Path


```

File: release.py [hunk 2]
```diff
@@ -2,5 +2,5 @@
 
 VERSION = "0.7.2"
 CODENAME = "Grey Edition"
-SAVEPATH = "/.cache/pokete"
+SAVEPATH = "/.local/share/pokete"
```
Comment (lxgr-linux) [inline release.py:5]:
```suggestion
SAVEPATH = os.environ.get("XDG_DATA_HOME", str(Path.home)+"/.local/share") + "/pokete"
```

**Review by lxgr-linux — CHANGES_REQUESTED** [review]
Since it was requested here #137, I think it's a good idea to use the `$XDG_DATA_HOME` environment variable to determine the save location.

**Comment by lxgr-linux** [comment]
@daynetran

File: release.py (hunk 1 above)
Comment (lxgr-linux) [reply release.py:2]: Your forgot to apply this

**Comment by MaFeLP** [comment]
Thank you for your contribution.

I would just like to have it fully backwards compatible, which means for me no user intervention required.
I would therefore love to see a check, if data in the `SAVEPATH` exists, and if not, try to load it from the old save location. If there also is no data to be found, start a 'fresh' instance of pokete.

**Comment by lxgr-linux** [comment]
Ready for review @MaFeLP

**Pull request merged**